### PR TITLE
Schedule Empty State

### PR DESCRIPTION
**Pull request by niamh-power**
### Create empty state 

- Add `nothing_here` image to assets
- Add empty state to schedule view as the default

### Screenshots

simulator screen shot - iphone 8 plus - 2018-03-16 at 17 30 48

### CODE REVIEW

**Review by rock3r — APPROVED** [review]
Ez! :100:

File: lib/schedule/schedule.dart [hunk 1]
```diff
@@ -17,8 +17,25 @@ class ScheduleView extends StatelessWidget {
...
+            new Text('Nothing here. Come back soon!',
+              textAlign: TextAlign.center,
+              style: theme.textTheme.display1.copyWith(color: Colors.black, fontSize: 16.0),),
+          ],
+      )
+    );
+
+    final body = new PageView(children: <Widget>[
+      emptyState,
+    ],);
```
Comment (fourlastor) [inline lib/schedule/schedule.dart:34]: is the comma intended? not sure if it compiles cause there's no CI yet 👅

**Review by fourlastor — APPROVED** [review]
Seems good to me! Not familiar with the syntax but if that compiles let's 🚢

File: lib/schedule/schedule.dart (hunk 1 above)
Comment (niamh-power) [reply lib/schedule/schedule.dart:34]: The trailing commas thing baffles me too, not entirely sure if it's needed, will check!

Comment (rock3r) [reply lib/schedule/schedule.dart:34]: It's needed for `dartfmt` and the IDE to know how to format lists correctly and is a common practice in Flutter code

**Pull request merged**